src/components/completing-party/CompletingParty.stories.ts

```diff
@@ -1,43 +1,19 @@
-import type { Meta } from '@storybook/vue'
-import { CompletingParty } from './index'
-import { required, maxLength } from 'vuelidate/lib/validators'
+import { Meta } from '@storybook/vue3'
+import CompletingParty from './CompletingParty.vue'
+import { AddressIF } from '@bcrs-shared-components/interfaces'
+import { DefaultSchema } from '../base-address/resources/default-schema'
```

Comment:
Should a normal component (in one of our UIs) be able to import the default schema? If so then it should be exported as part of Base Address. Or, if it's just used here for testing purposes then this is OK.

Reply:
Obsolete code. Resolving.